postgrator.json

```diff
@@ -0,0 +1,9 @@
+{
+    "migrationPattern": "migrations/*",
```

Comment:
What does postgrator do, what are migrations, why are they necessary?

Reply:
Postgrator is a migration library that allows us to control schema incrementally.

Migrations are necessary as schema changes over time and at times schema changes can be destructive. Using migrations with an incremental path allows us to more safely control how schema changes and as it's tied to the codebase the idea is that the latest migration should always be the current schema for any build.